cmd/arrai/command_util.go

```diff
@@ -28,3 +28,30 @@ func fetchCommand(args []string) string {
 	}
 	return ""
 }
+
+func insertRunCommand(globalFlags []cli.Flag, args []string) []string {
+	globalFlagsEndIndex := 1
+	globalFlagsMap := make(map[string]struct{})
+
+	for _, f := range globalFlags {
+		for _, n := range f.Names() {
+			globalFlagsMap[n] = struct{}{}
+		}
+	}
+
+	for i, a := range args[1:] {
+		if strings.HasPrefix(a, "-") {
+			name := strings.TrimLeftFunc(a, func(r rune) bool { return r == '-' })
```

Comment:
```suggestion
			name := strings.TrimLeft(a, "-")
```

Reply:
I needed to use the `TrimLeftFunc` so that it can trim both `-d` and `--debug`. Just using `TrimLeft` will only handle `-d`.

Reply:
nevermind, I was misled!!